packages/v4/patternfly-docs/content/developer-resources/accessibility-guide.md

```diff
@@ -14,57 +14,64 @@ When it comes to product experiences, no one should get left behind. That's why
 <br />
 
 ### Motor
-Users with poor motor control can use a range of devices to access contents. **
-- Users who rely on a keyboard need elements that are keyboard accessible and highly visible when in focus. 
+
+Users with poor motor control can use a range of devices to access contents.
+
+- Users who rely on a keyboard need elements that are keyboard accessible and highly visible when in focus.
 - Users who rely on a mouse or touch need target areas that are large enough to be hit easily.
 <br />
 
 ### No vision
-Users with no vision rely on screen readers to access web sites and applications. Often, screen reader users will navigate a page by browsing specific elements, like headers, links, or form elements. 
-- **Tip:** Use semantic elements and check that labels are meaningful when pulled out of context.    
 
+Users with no vision rely on screen readers to access web sites and applications. Often, screen reader users will navigate a page by browsing specific elements, like headers, links, or form elements.
+
+- **Tip:** Use semantic elements and check that labels are meaningful when pulled out of context.
 
 ### Low vision
-Users with low vision can have different needs depending on the nature of their visual impairment. Users may have difficulty with color differentiation, blurriness, or lack of vision in central or peripheral areas. 
-- **Tip:** These needs mean that interfaces should not rely on color to communicate information, palettes need to have sufficient contrast, and layouts should be responsive when font sizes are increased.
+
+Users with low vision can have different needs depending on the nature of their visual impairment. Users may have difficulty with color differentiation, blurriness, or lack of vision in central or peripheral areas.
+
+- **Tip:** These needs mean that interfaces should not rely on color alone to communicate information, palettes need to have sufficient contrast, and layouts should be responsive when font sizes are increased.
 <br />
 
-### Cognitive 
-Users who have difficulty processing information benefit from well-written content. 
-- Information should be clear, concise, and easy to scan. 
+### Cognitive
+
+Users who have difficulty processing information benefit from well-written content.
+
+- Information should be clear, concise, and easy to scan.
 - Consider visual hierarchy, chunk content into short, related sections, and avoid long paragraphs.
 <br />
 <br />
 
-These are some of the main user groups, but it is not an exhaustive list. Often “the Persona Spectrum”, coined by Microsoft, can be used to understand related mismatches and motivations across a spectrum of permanent, temporary, and situational scenarios. 
+These are some of the main user groups, but it is not an exhaustive list. Often “the Persona Spectrum”, coined by Microsoft, can be used to understand related mismatches and motivations across a spectrum of permanent, temporary, and situational scenarios.
```

Comment:
I think, according to the original issue, this line should be changed from `Often "the Persona Spectrum"` to `Often the "Persona Spectrum"`